src/internal/m365/collection/groups/handlers.go

```diff
@@ -67,3 +67,14 @@ type getItemer[I groupsItemer] interface {
 		itemID string,
 	) (I, *details.GroupsInfo, error)
 }
+
+// ---------------------------------------------------------------------------
+// Container management
+// ---------------------------------------------------------------------------
+
+type container[C graph.GetIDer] struct {
```

Comment:
I'm starting to lose sense of what `container` and the `folders` field mean in groups context. Could you please explain a bit? :)

Reply:
Thanks for the explanation. This helps!